api/src/api/healthCheckRouter.ts

```diff
@@ -0,0 +1,61 @@
+import { Router } from "express";
+
+/* TODO: Building out this feature:
+ *
+ * - Ensure continued access to the /health endpoint for the entire app.
```

Comment:
What more needs to be done to ensure this? I would think you've already finished this

Reply:
Just testing, which I haven't done yet.